docs/excel/co-authoring-in-excel-add-ins.md

```diff
@@ -39,10 +40,22 @@ If you want User A's custom visualizations to respond to changes made by coautho
 
 ## Caveats to using events with coauthoring
 
-As described earlier, in some scenarios, triggering events for all coauthors provides an improved user experience. However, be aware that in some scenarios this behavior can produce poor user experiences. 
+As described earlier, in some scenarios, triggering events for all coauthors provides an improved user experience. However, be aware that in some scenarios this behavior can produce poor user experiences.
 
 For example, in data validation scenarios, it is common to display UI in response to events. The [BindingDataChanged](/javascript/api/office/office.bindingdatachangedeventargs) event described in the previous section runs when either a local user or coauthor (remote) changes the workbook content within the binding. If the event handler of the `BindingDataChanged` event displays UI, users will see UI that is unrelated to changes they were working on in the workbook, leading to a poor user experience. Avoid displaying UI when using events in your add-in.
 
+## Avoiding table row coauthoring conflicts
+
+Updating a table can cause coauthoring conflicts. The following guidance should help you avoid issues (and that yellow bar Excel shows asking users to refresh):
+
+1. Use [`Range.values`](/javascript/api/excel/excel.range#values) instead of [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-). Setting the `Range` values directly below the table automatically expands the table. Otherwise, adding table rows through the `Table` APIs results in merge conflicts for coauth users.
```

Comment:
The Range.values and TableRowCollection.add links also appear broken to me (maybe it's an issue with the review site?)